salt/elife-metrics/init.sls

```diff
@@ -144,6 +144,15 @@ load-articles-every-day:
         - minute: 0
         - hour: 0
 
+# once a week, remove any partial files that are hanging around
+# these are deliberate cache misses for periods that will return partial results
+rm-partial-files-every-week:
+    cron.present:
+        - user: {{ deploy_user }}
+        - name: cd /src/elife-metrics/output && find . -name '*\.partial' -delete
+        - identifier: rm-partial-files-every-week
+        - special: "@weekly"
```

Comment:
would you like to be more precise here and say `Monday at 10 am` or similar, or is it really not important as it's just a matter of disk space?

Reply:
I'm not sure the `.partial` files have ever been deleted, it's just house keeping really.

It just occurred to me while going through the code that they must be accumulating, and lo ...